src/app/home-page/home-logged-in/home-logged-in.component.html

```diff
@@ -13,10 +13,148 @@
   ~    See the License for the specific language governing permissions and
   ~    limitations under the License.
   -->
-
 <div fxLayout>
   <app-my-sidebar fxFlexAlign="stretch"></app-my-sidebar>
 
+  <div id="row-holder" fxLayout="column" fxLayoutAlign="start start" class="p-3 w-100">
+    <mat-card class="alert alert-info mx-1 w-100" role="alert"> Major Release: </mat-card>
```

Comment:
I see it in the mock, but I'm not sure what the intent is there. Is it to display the latest news item, or is it to figure out the latest major release (which we don't have an API for?). Depending, you may not want to hard-code `Major Release:` here.

Reply:
I think this was a placeholder for what because our current "news and events"
So I think "news and events" can replace this